addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2513,6 +2513,55 @@ QUnit.test('chat window should remain folded when new message is received', asyn
     );
 });
 
+QUnit.test('with conversation scrolled to end, when composer size changes, conversation should still be scrolled to end', async function(assert) {
+    assert.expect(2);
+
+    // channel that is expected to be found in the messaging menu
+    // with random unique id, needed to link messages
+    this.data['mail.channel'].records.push({ id: 20 });
+    for (let i = 0; i < 10; i++) {
```

Comment:
would be worth mentioning that enough messages are needed to have a scrollbar in chat window, and how the value has been decided (height of chat window vs height of message * X)